DESCRIPTION

```diff
@@ -10,6 +10,7 @@ Authors@R: c(
   person("Barret", "Schloerke", role="aut", email="barret@rstudio.com"),
   person("Frans", "van Dunné", role="ctb", email="frans@ixpantia.com"),
   person("Sebastiaan", "Vandewoude", role="ctb", email="sebastiaanvandewoude@gmail.com"),
+  person("Mauricio", "Vargas", role="ctb", email="mvargas@dcc.uchile.cl"),
```

Comment:
Please add news item tagging yourself and the PR
```suggestion
```

Please also add a news item for #398